### ISSUE TITLE
[BUG] Shouldn't show the message (No transactions yet on Snort) 

### ISSUE THREAD

**Issue by bitcoinuser** [getAlby/lightning-browser-extension#2219]
### Describe the bug

During the loading process show the message (No transactions yet on Snort) . The message should appear only after the loading process be finished and really no transactions exist.

See the video:

https://user-images.githubusercontent.com/1178856/225382187-2a1dcf1b-88e5-4e25-8aaa-60dd7cb085db.mp4

### Steps To Reproduce

1. Go to a site that you have lightning transactions.

### Expected behavior

The message  (No transactions yet on Snort) should appear only after the loading process be finished and really no transactions exist.

### Alby information

Alby: 1.28.1

**Comment by SumantxD** [comment]
@bitcoinuser could you assign this issue to me. i would love to work on it.

**Comment by Ladyperpetua** [comment]
@can i please be assigned this issue @reneaaron

**Comment by mviswanathsai** [comment]
I have solved this, can i push a PR?

**Comment by reneaaron** [comment]
Can you submit a pull request please?

**Comment by mviswanathsai** [comment]
@reneaaron I have changed code, but I am unable to test it without transactions. How can I do this?

**Comment by mviswanathsai** [comment]
@reneaaron sure, could you quickly clarify from where we get the message "no transactions on **snort** yet". I found out the message displayed on default page is from the `\src\i18n\locales\en\translation.json`. Wondering where the message for snort is from

**Comment by mviswanathsai** [comment]
@reneaaron sure, I just need to be able to test it. How can I use snort on testnet?

**Comment by mviswanathsai** [comment]
@reneaaron I have created a PR, https://github.com/getAlby/lightning-browser-extension/pull/2323

**Comment by reneaaron** [comment]
@mviswanathsai It's closed now? :thinking:

**Comment by mviswanathsai** [comment]
@reneaaron Yeah, I messed up the PR, tests were failing. I will open a new PR now. 
Sorry, very newbie of me.

**Comment by mviswanathsai** [comment]
@reneaaron here is the PR, #2324

**Comment by mviswanathsai** [comment]
@reneaaron is the PR good?
